### PR TITLE
Update Application Insights sample discoverability

### PR DESCRIPTION
**Pull request by boblangley**
This PR:

- Adds tags / related to the meta data for the sample.
- Moves the sample under `samples/logging` from the root to make it more discoverable.

### CODE REVIEW

File: samples/logging/application-insights/sample.md [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
 title: Report metrics to Azure Application Insights
 component: Metrics
-reviewed: 2017-07-29
+reviewed: 2017-10-06
+tags:
+- Azure
+- Metrics
+related:
+ - nservicebus/operations/metrics
+ - samples/azure/
```
Comment (boblangley) [inline samples/logging/application-insights/sample.md]: Not sure if `samples/azure` is appropriate here. I would be nice if this sample could be listed under both `Logging and Metrics` and `Azure`

File: samples/logging/application-insights/sample.md [hunk 2]
```diff
@@ -1,7 +1,16 @@
...
+summary: Shows how to report NServiceBus metrics data to Azure Application Insights
 component: Metrics
-reviewed: 2017-07-29
+reviewed: 2017-10-06
+tags:
+- Azure
+- Metrics
+related:
+ - nservicebus/operations/metrics
+ - samples/azure
```
Comment (boblangley) [inline samples/logging/application-insights/sample.md]: Not sure if `samples/azure` is appropriate here. I would be nice if this sample could be listed under both `Logging and Metrics` and `Azure` - commented again after fixing the path

File: samples/logging/application-insights/sample.md [hunk 3]
```diff
@@ -1,7 +1,16 @@
 ---
 title: Report metrics to Azure Application Insights
+summary: Shows how to report NServiceBus metrics data to Azure Application Insights
 component: Metrics
-reviewed: 2017-07-29
+reviewed: 2017-10-06
+tags:
+- Azure
+- Metrics
```
Comment (DavidBoike) [inline samples/logging/application-insights/sample.md]: Content verification says "Each tag must have more than one page" ... guessing Metrics is the one that has only one. Add it to other related pages or remove the tag.

File: samples/logging/application-insights/sample.md (hunk 2 above)
Comment (DavidBoike) [reply samples/logging/application-insights/sample.md]: Not sure how that will render. I think best course is to get this merged and see how it shows up and then adjust.

**Pull request merged**